lib/double_entry/reporting/aggregate_array.rb

```diff
@@ -50,8 +53,9 @@ def retrieve_aggregates
         scope = LineAggregate.
                 where(:function => function).
                 where(:range_type => 'normal').
-                where(:account => account.to_s).
-                where(:code => code.to_s).
+                where(:account => account.try(:to_s)).
+                where(:code => code.try(:to_s)).
+                where(:partner_account => partner_account.try(:to_s)).
```

Comment:
It needs to find the aggregates without partner_account (and similarly for account and code), doesn't it? They will contain the total figures across all partner_accounts, right?